test/data/test_builtin_datasets.py

```diff
@@ -51,6 +51,9 @@ def test_wikitext2(self):
         conditional_remove(cachefile)
 
         train_dataset, valid_dataset, test_dataset = WikiText2()
+        train_dataset.data = torch.cat(tuple(filter(lambda t: t.numel() > 0, train_dataset.data)))
```

Comment:
I don't think the .data field is necessary for this and we should probably even consider making it private.